source/includes/entityref/paybill/_expensesheet.md

```diff
@@ -0,0 +1,146 @@
+# ExpenseSheet
+
+This entity represents the required certifications for a Job Submission.
+
+
+
+<table>
+    <colgroup>
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+        <col width="20%" />
+    </colgroup>
+    <thead>
+        <tr class="header">
+            <th>ExpenseSheet fields</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Not null</th>
+            <th>Read-only</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="even">
+            <td>id</td>
+            <td>Integer</td>
+            <td>Unique identifier for this entity.</td>
+            <td>X</td>
+            <td>X</td>
+        </tr>
+        <tr class="odd">
+            <td>addedAtUtc</td>
+            <td>Timestamp</td>
+            <td>Record of when Timesheet was created</td>
+            <td>X</td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>addedByUser</td>
+            <td>To-one association</td>
+            <td>CorporateUser default fields: id, firstName, lastName</td>
```

Comment:
See examples in existing doc. Want something like this throughout:
http://bullhorn.github.io/rest-api-docs/entityref.html#pay-and-bill-billmastersurchargerate
see source for that for how bullets are done